Refining the README introduction (#8)
runlyio/core-dotnet: README.md

> Runly is an opinionated framework for writing multi-threaded jobs. You focus on writing your business logic and Runly gives you a CLI, multi-threading, retries, and more out-of-the-box.

Reviewer: ```suggestion
Runly is a toolset for simplifying complex applications by writing compartmentalized, testable code. You focus on writing your business logic and Runly gives you a CLI, multi-threading, retries, and more out-of-the-box.
```

Author: I think we should leave "opinionated framework" in there. It conveys that we've though of the issues you are going to have before you have and have designed the API in a way that will give you the least friction ([stolen from here](https://stackoverflow.com/questions/802050/what-is-opinionated-software#:~:text=An%20opinionated%20framework%20is%20one,made%20by%20the%20framework%20designer.%20%E2%80%93)). Maybe we could spell that out more explicitly if you want?

Reviewer: I know this is a thing, but really everyone has an opinion that shows in their code. I feel like we don't need to hit people over the head with this, it should be obvious, and I wouldn't want to turn people off right away who look at opinionated software as bad software.

See [more in-depth examples](./examples) or [start reading the docs](https://www.runly.io/docs/).

## Why Write Code Like This?

> A lot of problems that us software developers are tasked with solving everyday boil down to processing lists of items. Some examples could be:

Reviewer: ```suggestion
A lot of problems that software developers are tasked with solving everyday boil down to processing lists of items. Some examples could be:
```

> * importing and processing records from large uploaded CSV files into a database

Reviewer: ```suggestion
* importing and processing records from large files into a database
```

> * downloading or uploading import/export files automatically on a schedule

Reviewer: ```suggestion
* downloading or uploading import/export files on a schedule automatically
```

* queueing and sending large amounts of transactional emails
* generating PDFs from a list of records in a database
* handling ETL scenarios

A Runly job takes this pattern and standardizes it into a base `Job` class. Building and running jobs in a predictable way like this allows us to build lots of goodies on top of your core jobs. Things like multi-threading, scaling work across different machines, and resiliency and fault-tolerance become almost trivial for you to implement.

In fact, we have done exactly that and built a whole platform on top of this core OSS project.

## The Runly Platform

> You can turbocharge your job apps using the [Runly Platform](https://www.runly.io/platform/). The Runly Platform is a SaaS which helps shepherd your jobs from deployment to monitoring and scaling in production.

Reviewer: ```suggestion
You can turbocharge your job apps using the [Runly Platform](https://www.runly.io/platform/). The Runly Platform is SaaS which helps shepherd your jobs from deployment to monitoring and scaling in production.
```

> * Automatically deploy your jobs to your machines or the cloud as NuGet packages using tools you already know and love.

Reviewer: ```suggestion
* Automatically deploy your jobs to your existing machines or the cloud as NuGet packages using tools you already know and love.
```

* Use familiar REST concepts to queue and schedule background jobs from your web or mobile app.
* Easily integrate progress bars and job status into your app using [Runly UI](https://www.runly.io/ux/).
* Use CRON expressions for fine-grained control of job scheduling.
* Get insights into job failures and performance issues before your users notice.

You can get started on a [generous free tier](https://www.runly.io/pricing/) with no commitment and no credit card. [Get started for free using your GitHub account](https://www.runly.io/dashboard/) or learn more about [how it works](https://www.runly.io/docs/).

## Is this project sustainable?

> [Sustainability in OSS is a problem](https://twitter.com/UdiDahan/status/1282925432166285312). We are trying to work on this project full-time and in order to do that, we have to feed our families. We think providing value via a paid service is the best way to do that while still solving problems for people with the core OSS offering.

Reviewer: ```suggestion
[Sustainability in OSS is a problem](https://twitter.com/UdiDahan/status/1282925432166285312). We are working on this project full-time and in order to keep doing that, we have to feed our families. We think providing value via a paid service is the best way to do that while still solving problems for people with the core OSS offering.
```

We are fiercely independent and not going anywhere. [Read more about our philosophy](https://www.runly.io/about/).

> Our mission is to create great developer experiences making it trivial to build and scale jobs while providing a great user experience in your app. We have plenty of ideas on how to do this but we need your support to continue working on this project.

Reviewer: ```suggestion
Our mission is to create great developer experiences by making it trivial to build and scale jobs in order to provide a great user experience in your app. We have plenty of ideas on how to do this but we need your support to continue working on this project.
```
